tests/e2e/k8s/test-pod.yaml.tpl

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  containers:
+    - name: nginx
```

Comment:
This looks odd. The container is named `nginx` but the image is `busybox`

Reply:
fixed